src/app/app-config.service.ts

```diff
@@ -96,38 +169,42 @@ export class AppConfigService {
             alwaysShow: ['license', 'url']
           },
           '/persistent_identifiers/items': {
-            alwaysShow: ['type', 'value']
+            alwaysShow: ['value'],
+            order: ['value', 'schema', 'material', 'source']
           },
           '/publication_info/items': {
-            alwaysShow: ['journal_title', 'journal_volume', 'journal_issue', 'artid', 'cnum', 'year', 'confpaper_info']
-          },
-          '/references': {
-            longListNavigatorConfig: {
```

Comment:
@StellaCh are you sure about this delete because this is for having pagination and search for references?